data/store/scripts/outfits.lua

```diff
@@ -0,0 +1,79 @@
+-- Load outfits
+local outfits = {}
+
+for line in io.lines('data/XML/outfits.xml') do
```

Comment:
Or even better an Outfit class so we don't have to send huge tables when we might just want the sex of a citizen outfit.